Resources/Prototypes/Corvax/Loadouts/role_loadout.yml

```
@@ -0,0 +1,23 @@
+
+- type: roleLoadout
+  id: JobMedSyn
+  groups:
+  - SmugglerCaptainHead
+  - MedSynJumpsuit
+  - SmugglerBags
+  - SmugglerGloves
+  - MedSynGlasses
+  - SmugglerShoes
+  - SmugglerNeck
+  - SmugglerOuterclothing
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)